5-Heroku/server/models/todoModel.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose')
+const todoScheme = new mongoose.Schema({
+    todoText: {
+        type: String,
+        required: true
+    },
+    todoId: {
+        type: String,
+        required: true
+    },
+    checked: {
+        type: Boolean,
+        required: true
+    },
+    userId: {
+        type: String,
+        required: true
+    }
+})
+
+const Todo = mongoose.model('Todo', todoScheme)
+
+const modelGetTodoById = async (req, res) => {
```

Comment:
TodoModel should be logic only it shouldn't know anything about requests / responses / etc..
this should be handled in the `controller` in your case the controller is your `todoController.js`

I would expect TodoModel to implement a logical interface something like
```
class TodoModel {
   getItems(userId) {
      //do some logic
      return items
   }
}
```